v22.2/sso-sql.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Cluster Single Sign-On (SSO) for Self-hosted CockroachDB
```

Comment:
The same issue on the page formatting is happening here. That is, the sidebar is pushed to one side weirdly. I *think* it's to do with the heading hierarchy

Reply:
Whoops, just fixed it. It was an improperly formatted 'callout', @nickvigilante helped me figure it out!

Reply:
Ahhh nice! Good to know!